ash/src/entry.rs

```diff
@@ -264,10 +264,10 @@ impl Entry {
         &self.entry_fn_1_1
     }
 
-    #[deprecated = "This function is unavailable and therefore panics on Vulkan 1.0, please use `try_enumerate_instance_version` instead"]
+    #[deprecated = "This function is unavailable and therefore panics on Vulkan 1.0, please use `try_enumerate_instance_version()` instead"]
```

Comment:
I thought (intradoc) links were also possible in deprecation blocks (similar to other markdown), but nope :(

Reply:
Aw, that seems like a really useful place for them too. Feature request upstream?

Reply:
Heh feature requests have to go to the "internals" forum, I'll see what I can do.